### ISSUE TITLE
TypeError: Str is not a constrcutor

### ISSUE THREAD

**Issue by MichaelBrink** [LedgerHQ/ledgerjs#548]
Getting a typeerror when invoking the Str constructor from @ledgerhq/hw-app-str (trying to execute a few basic stellar-sdk command)

node -v -> v.14.15.1
"@ledgerhq/hw-app-str": "^5.34.0",
"@ledgerhq/hw-transport-u2f": "^5.34.0",
"stellar-sdk": "^7.0.0"

code snippit:

```
import { default as TransportU2F } from "@ledgerhq/hw-transport-u2f";
import {default as Str} from "@ledgerhq/hw-app-str";
import StellarSdk, { Networks } from "stellar-sdk";
 
const getStrAppVersion = async () => {
    const transport = await TransportU2F.default.open();
    const str = new Str(transport);
    const result = await str.getAppConfiguration();
    return result.version;
} 

getStrAppVersion().then(v => console.log(v));
```
The error I am getting is:
`(node:28256) UnhandledPromiseRejectionWarning: TypeError: Str is not a constructor
`

Any guidance would be appreciated, ty

**Comment by gre** [comment]
can you try to do

```
import Str from "@ledgerhq/hw-app-str";
```

?

**Comment by MichaelBrink** [comment]
Yeah had that initially, but same err response (same with TransportU2F), this doesn't seem to make a difference

**Comment by yul761** [comment]
@gre I am sitting with a similar 'Not a constructor' issue. Could it be related to the ES6 module?

**Comment by MichaelBrink** [comment]
> can you try to do
> 
> ```
> import Str from "@ledgerhq/hw-app-str";
> ```
> 
> ?

@gre would appreciate any other suggestions, please

**Comment by MichaelBrink** [comment]
@gre  Resolved this by using webpack to compile  - this dealt with the CJS vs ES export/import inconsistency

**Comment by gre** [comment]
Ok. Good to know, we will need to improve this in future.
